backend/api/Database/Models/Mission.cs

```diff
@@ -53,10 +53,9 @@ public MissionStatus MissionStatus
         [Required]
         public MissionMap Map { get; set; }
 
-        [Required]
-        public DateTimeOffset StartTime { get; set; }
+        public DateTimeOffset? StartTime { get; set; }
```

Comment:
```suggestion
        [Required]
        public DateTimeOffset StartTime { get; set; }
```

Reply:
Maybe we should rename this to `ScheduledStartTime`, and add another field `StartTime` that represents the actual time the mission started.